scripts/biomedical/mesh/mesh_record_test.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+Author: Suhana Bedi
+Date: 08/29/2022
+Name: mesh_record_test.py
+Description: runs unit tests for format_mesh_record.py
+Run: python3 mesh_record_test.py
+'''
+import unittest
+from pandas.testing import assert_frame_equal 
+from format_mesh_record import *
+
+class TestParseMesh(unittest.TestCase): 
+    """Test the functions in format_mesh_record.py"""
+
+    def test_main(self):
+        """Test in the main function"""
+        # Read in the expected output files into pandas dataframes
+        df1_expected = pd.read_csv('unit-tests/mesh_record_test.csv')
```

Comment:
Is there a reason why these test files are so big? Can you please limit it to ~10 entities for each? They currently are not viewable on GitHub.